clang/test/CodeGenHLSL/res-may-alias.hlsl

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -res-may-alias -finclude-default-header -triple dxil-pc-shadermodel6.3-library -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefix=FLAG
```

Comment:
did you want to change these to `clang_dxc` aswell since you are making a dxc driver change?

Reply:
Yea I will change these to clang_dxc